test/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+
+add_executable(testApp test.cpp
+                  ../src/engine/universe_master.cpp
```

Comment:
i think
```cmake
add_executable(testApp
    test.cpp
    ../src/engine/universe_master.cpp
)
```
as in `./src/engine/CMakeLists.txt` would be slightly better

Reply:
Removed it in 1fa21603356c4e4625bab7f8ad4667367d9af987, was legacy code that should no longer be there

Reply:
okey, no problem :ok_hand: